resources/META-INF/plugin.xml

```diff
@@ -119,7 +119,6 @@
                      groupName="Pants" enabledByDefault="true"
                      implementationClass="com.twitter.intellij.pants.inspection.PythonPluginInspection"/>
 
-    <compileServer.plugin classpath="intellij-pants-plugin-publish.jar"/>
```

Comment:
We used to just throw an error if we reached external build process by accident, and now that's been moved to `PantsBuildProcessParametersProvider`, so no need for this anymore.